src/_i18n/en.yml

```diff
@@ -775,3 +713,13 @@ service-line-example:
   curated-publications:
     intro-title: Example of publications section
     intro-description: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+all-service-lines:
+  page-sections:
+    how-we-solve-your-problems: how-we-solve-your-problems
+    how-we-help-our-clients: how-we-help-our-clients
+    testimonials: testimonials
+    compass: compass-by-codurance
+    webinar: webinar
+    publications: publications
+    get-in-touch: get-in-touch
```

Comment:
Looks like these need to be updated in the HTML as well. Missing commit perhaps?

Reply:
I've updated the HTML to now reference these.